### DISCUSSION TITLE
How to run camel-karavan source code in visual studio

### DISCUSSION THREAD

**Discussion by MN-Dheeraj** [apache/camel-karavan#1269] [Q&A]
### Ask a question

I see this page for instructions to build and run the source code of camel karavan.
[[[How can I create a new REST service to get data from Postgres database](https://github.com/apache/camel-karavan/blob/main/docs/DEV.md](https://github.com/apache/camel-karavan/blob/main/docs/DEV.md)](https://github.com/apache/camel-karavan/blob/main/docs/DEV.md#to-run-karavan-app-in-the-local-machine-for-debugging)

I imported the source code into my Ubuntu system and started the application. Now, the karavan-UI is up and running.
However, I would like to know if the source code can be started(run) in Visual studio. This is to run the application(source code) in Visual studio and add my debug points in the Vscode and check the flow of various use-cases.

### Variant

Vs Code Extension

### Container Management (if applicable)

None

### Operating System (if applicable)

Windows

### Version

4.5.0

**Comment by kondetiramanakumar** [comment]
@davsclaus could you provide your input for this???

**Comment by kondetiramanakumar** [comment]
@davsclaus Highly Appreciated if you could answer my query?
